component-library/component-lib/src/lib/form-components/input/input.component.html

```diff
@@ -26,7 +26,7 @@
             [type]="typeControl"
             [id]="config.id"
             [formControlName]="config.id"
-            [attr.aria-invalid]="formControl?.errors"
+            [attr.aria-invalid]="formControl?.invalid"
```

Comment:
Check to make sure this works in Safari

Reply:
![Screen Shot 2023-05-11 at 08 28 20 AM  1](https://github.com/IRCC-ca/ds-sdc-dev/assets/111757720/3000ef5d-3501-4520-bbe4-322402f46c8c)
Confirmed safari is working